### ISSUE TITLE
Exception in distributed configuration: com.orientechnologies.common.exception.OException: Cannot find best cluster for class

### ISSUE THREAD

**Issue by shlapkov** [orientechnologies/orientdb#4437]
In 2.0.10 and 2.0.11 we are getting exception in distributed configuration: com.orientechnologies.common.exception.OException: Cannot find best cluster for class looks like issue happens only on Linux (Red hat, CentOS), Steps to reproduce:

1. We are setting up a two node cluster.
2. We start the node1 and then start the node2. Now the cluster is up.
3. Attempt save any records on node1 failed. We get the following exception:

com.orientechnologies.common.exception.OException: Cannot find best cluster for class 'platform-DataflowContents' on server 'node_SI-RH6-Cluster101.pbi.global.pvt_8080'. ClusterStrategy=local

**Comment by AnthonyHebig** [comment]
did you drop your class on node1 ? (so it would also drop the node1 cluster for this class)

maybe you can do "ALTER CLASS <yourclass> ADDCLUSTER <some_cluster_name>" and try to save new records on node1 again

**Comment by shlapkov** [comment]
We don't this before prior 2.0.x every class had only one cluster. How there are multiply clusters per node and it doesn't seems to be working correctly. Looks like it is related to order in server list: "servers": ["nodel","node2","<NEW_NODE>"] ... but I am not sure.

**Comment by shlapkov** [comment]
Issue appears to be related to the fact that we are setting server names as: "servers": ["node_SI-RH6-Cluster102.pbi.global.pvt_8080","node_SI-RH6-Cluster103.pbi.global.pvt_8080","<NEW_NODE>"]. Once we updated names to: "servers": ["node_Cluster102_8080","node_Cluster101_8080","<NEW_NODE>"] issue was fixed. Is there any regulations for server names in Orient DB? Why we are getting exception: "Cannot find best cluster for class" and not some IO error.
